compile_docker.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+
+BASE_IMAGE="ubuntu:jammy-20220421"
+
+IMAGE_NAME="scons_image"
+CONTAINER_NAME="scons_container"
+
+COMPILER_DIR=embedded/platforms/toolchain-gccarmnoneeabi/
+
+# wrapper variables 
+COMPONENT=""
+
+# pull base image 
+docker pull ${BASE_IMAGE}
+	
+# build base ubuntu image  
+docker build -t ${IMAGE_NAME} .
+
+docker-compose up -d
+
+# download compiler 
+
+# search recursively through repo dir for compiler 
+find . -name "gcc-arm-11.2-2022.02-x86_64-arm-none-eabi*" > compiler_exist.txt
```

Comment:
why? The compiler has to exist in one specific place for this whole thing to work, no need to check elsewhere